tests/test_config.py

```diff
@@ -13,34 +13,35 @@
 class TestConfig:
     @pytest.fixture(autouse=True)
     def _init_tmpdir(self, tmpdir):
+        os.environ["HOME"] = str(tmpdir)
         self.tmpdir = tmpdir
 
-    def _assert_defaults(self, cfg):
+    def _assert_defaults(self, cfg: config.Config):
         assert cfg.general is not None
         assert cfg.general.log_level == "INFO"
 
         assert cfg.chat is not None
-        assert cfg.chat.model == "models/merlinite-7b-lab-Q4_K_M.gguf"
+        assert cfg.chat.model == config.DEFAULTS.DEFAULT_MODEL
```

Comment:
I recommend that you leave the hard-coded values here / replace them with correct new hard-coded values. For tests it is sometimes a good idea to violate the DRY dogma. It makes it easier to detect bugs.

Reply:
+1 Keeping test constants separate and not linked to impl constants is good advice (although ok for tests to share between themselves)

Reply:
That makes sense, I'll fix this.